src/main/java/edu/harvard/iq/dataverse/api/Meta.java

```diff
@@ -178,8 +174,6 @@ public String dataset(@PathParam("datasetId") Long datasetId, @QueryParam("exclu
             throw new ServiceUnavailableException();
         }
 
-        response.setHeader("Access-Control-Allow-Origin", "*");
-
```

Comment:
In 8c7fb34 we flagged parts (or all?) of this API as deprecated and there's a test that calls `getMetaDatafileDeprecated` which seems to exercise the `@Path("datafile/{fileId}")` method. I don't know if `/api/meta` is still used by anything. I don't believe it's even documented. And Meta.java does not extend AbstractApiBean so I don't think it has access to allowCors or any other methods in there. Since CORS is now (in this pull request) enabled in ApiBlockingFilter, this "meta" API will still be subject to the new ":AllowCors" setting, right?

Reply:
Yes, it will.